requirements.txt

```
@@ -2,3 +2,4 @@ signac>=1.0.0
 jinja2>=2.8
 cloudpickle
 deprecation>=2
+tqdm>=4.10.0
```

```suggestion
tqdm~=4.10.0
```
[The tqdm package follows semantic versioning](https://tqdm.github.io/contributing/#semantic-versioning). Should we therefore maybe use the [compatible release](https://www.python.org/dev/peps/pep-0440/#compatible-release) specifier instead?

---

@csadorf Our use of tqdm is essentially the simplest use case. It looks like v5 is in progress and I don't think they're going to break `from tqdm import tqdm` according to the list of breaking changes. https://github.com/tqdm/tqdm/milestone/8

Usually I would say compatible versions are preferable, but I don't think we need to worry about that here. If you feel strongly about that, feel free to make the change here and in the signac repo (that PR was already merged).

---

Fine, no need for changes.